src/core/ActionAtomistic.h

```diff
@@ -42,6 +42,7 @@ class ActionAtomistic :
 
   std::vector<AtomNumber> indexes;         // the set of needed atoms
   std::set<AtomNumber>  unique;
```

Comment:
@carlocamilloni I would add a comment here:
/// unique should be an ordered set since we later create a vector containing the corresponding indexes

To avoid that in the future we replace it by mistake with an unordered_set